Sources/TuistKit/Commands/Account/AccountChangeUsernameCommand.swift

```diff
@@ -0,0 +1,34 @@
+import ArgumentParser
+import Foundation
+import Path
+
+struct AccountChangeUsernameCommand: AsyncParsableCommand {
```

Comment:
What do you think if we rename the command to `update` and update the attributes that are passed by flags? E.g.:

```bash
// Update only handle
tuist account update --handle foo

// Update handle and email
tuist account update --handle bar --email pedro@tuist.dev
```

Reply:
We can make the account handle optional from [this suggestion](https://github.com/tuist/tuist/pull/7209#discussion_r1901008626) to allow for this API. I'd still allow to pass the handle that you want to update in case you want to update a name of an organization. If the handle is not passed, we can make the assumption that the developer wants to update the handle of the account they are currently authenticated with.